docs/build-speed.md

```diff
@@ -0,0 +1,200 @@
+---
+id: build-speed
+title: Speeding up your Build
+---
+
+Building your React Native app could be **expensive** and take several minutes of developers time.
+This can be problematic as your project grows and generally in bigger organizations with multiple React Native developers.
+
+With [the New React Native Architecture](/docs/next/new-architecture-app-modules-android), this problem is becoming more critical
+as you might have to compile some native C++ code in your project with the Android NDK in addition to the native code already necessary for the iOS and Android platforms.
+
+To mitigate this performance hit, this page shares some suggestions on how to **improve your build time**.
+
+## Android
+
+### Build only one ABI
+
+When building your android app locally, you build all the 4 ABIs by default: `armeabi-v7a`, `arm64-v8a`, `x86` & `x86_64`.
```

Comment:
```suggestion
When building your android app locally, by default you build all the 4  Application Binary Interfaces (ABIs) : `armeabi-v7a`, `arm64-v8a`, `x86` & `x86_64`.
```
nit to clarify what ABI means

Reply:
we could even link to https://developer.android.com/ndk/guides/abis but no strong opinion